test/unit/test_cmd.c

```diff
@@ -288,8 +288,8 @@ bool test_cmd_help(void) {
 	r_cmd_desc_oldinput_new (cmd, p_cd, "px", px_handler, &px_help);
 
 	const char *p_help_exp = "Usage: p-usage   # p summary\n"
-		"| pd <num>                 # pd summary\n"
-		"| px <verylongarg_str_num> # px summary\n";
+		"| pd <num>                    # pd summary\n"
+		"| px[?] <verylongarg_str_num> # px summary\n";
```

Comment:
can this verylongarg string be shorter?

Reply:
what do you mean? this is a test to make sure the summaries (the text on the right) are aligned. What's the problem with having a long string in the tests??

Reply:
oh nvmits just a unit test